### PR TITLE
OH2-70 | Update app state after form submissions

### PR DESCRIPTION
**Pull request by SteveGT96**

### CODE REVIEW

**Review by axelpezzo — CHANGES_REQUESTED** [review]
@SteveGT96 This should be done for the Patient Form too. 

Beside, be sure to pass (and check if you receive it too) the lock property on the Patient DTO.

File: src/components/accessories/patientExams/PatientExams.tsx [hunk 1]
```diff
@@ -65,6 +66,7 @@ const PatientExams: FC = () => {
       dispatch(deleteLabReset());
       setShouldResetForm(true);
       setShouldUpdateTable(true);
+      dispatch(getLabsByPatientId(patientData?.code));
```
Comment (axelpezzo) [inline src/components/accessories/patientExams/PatientExams.tsx:69]: I think that this is useless now isnt' it?

File: src/components/accessories/patientNewBill/PatientNewBill.tsx [hunk 2]
```diff
@@ -100,6 +101,7 @@ const PatientNewBill: FC = () => {
       scrollToElement(infoBoxRef.current);
     }
     if (status === "SUCCESS") {
+      dispatch(getPendingBills(patient?.code ?? 0));
```
Comment (axelpezzo) [inline src/components/accessories/patientNewBill/PatientNewBill.tsx:104]: I think that this is useless now isnt' it?

File: src/components/accessories/patientVisit/patientVisit.tsx [hunk 3]
```diff
@@ -69,6 +69,9 @@ const PatientVisit: FunctionComponent = () => {
       scrollToElement(infoBoxRef.current);
       setShouldResetForm(false);
     }
+    if (changeStatus === "SUCCESS") {
+      dispatch(getVisits(patient?.code ?? 0));
+    }
```
Comment (axelpezzo) [inline src/components/accessories/patientVisit/patientVisit.tsx:74]: Here too

File: src/components/accessories/patientExams/PatientExams.tsx (hunk 1 above)
Comment (SteveGT96) [reply src/components/accessories/patientExams/PatientExams.tsx:69]: You're right !

**Review by alefalezza — CHANGES_REQUESTED** [review]
Please resolve conflicts

**Comment by alefalezza** [comment]
Please resolve conflicts

**Review by alefalezza — CHANGES_REQUESTED** [review]

File: src/components/accessories/patientNewBill/PatientNewBill.tsx [hunk 4]
```diff
@@ -210,3 +210,6 @@ const PatientNewBill: FC = () => {
   );
 };
 export default PatientNewBill;
+function getPendingBills(arg0: number): any {
```
Comment (alefalezza) [inline src/components/accessories/patientNewBill/PatientNewBill.tsx:213]: Remove

File: src/state/laboratories/reducer.ts [hunk 5]
```diff
@@ -1,4 +1,5 @@
 import produce from "immer";
+import { LaboratoryDTO } from "../../generated";
```
Comment (alefalezza) [inline src/state/laboratories/reducer.ts:2]: Remove unused import

**Review by alefalezza — CHANGES_REQUESTED** [review]

**Pull request merged**